src/setup/node.rs

```diff
@@ -56,8 +57,12 @@ pub enum NodeType {
 }
 
 pub struct NodeBuilder {
+    /// Node's startup configuration
     conf: NodeConfig,
+    /// Node's process metadata read from Ziggurat configuration files
     meta: NodeMetaData,
+    /// Pool of available stateful node indices which represent stateful nodes' directories
+    stateful_nodes_pool: Vec<usize>,
```

Comment:
unless we want to be able to pick some specific ones (e.g. 0 and 2), it would suffice for this value to just be a `usize`, no?

Reply:
```
        // We always expect to get indices in increasing order when popping the vector.
        let stateful_nodes_pool = (0..STATEFUL_NODES_COUNT).rev().collect::<Vec<usize>>();
```
This above happens below.

We need to ensure that the first provided stateful node starts from directory .ziggurat/ripple/stateful/1/
since that node contains some specific ledger info. But I'm thinking now, does that matter? I guess it should work with all three. 🤔

Reply:
yeah, and I've seen that the pool gets `pop`ped, but one could just as well subtract from `STATEFUL_NODES_COUNT`

Reply:
Yeah, the tests don't seem to like nodes 1 and 2. I'm gonna check more. Maybe it has something to do with the RPC IP address.

Reply:
Yep, the issue was with the RPC address `127.0.0.1:5005`, and other nodes use `127.0.0.2:5005`, `127.0.0.3:5005`

Done